analysis_for_grading.R

```diff
@@ -0,0 +1,157 @@
+#github was very uncooperative, we all did pretty equal work though 
+#1)our figures provide good evidence based on the cases per day the the disease originated in country X. 
+#2)We also determined that a vaccine developed in country Y would have limited uses for country X. 
+#The disparity in markers present between the two countries means that the little overlap that doesoccur would not confer good immunity to country X people. 
```

Comment:
Which graph you are referring to?
It is not clear how disparity it is. You need to point out the evidence from the graph
-0.5